project_proposal/supervisor_contract.md

```diff
@@ -1,23 +1,19 @@
-# Vejlederaftale
-- Der skal laves en gennemgang og dagsorden til hvert vejledermøde. Det skal 
-sendes til vejleder senest onsdag kl. 23.59.
-- Vejleder læser alle dokumenter og materialer igennen, vi sender, og dette 
-skal gøres før vejledermødet.
+# Supervisor agreement
+- For each supervisor meeting an agenda should be written and made available
+to the supervisor before 23:59 on Wednesday. 
+- It's assumed that the supervisor reads through the sent material before each
+meeting.
+- The supervisor will get access to our project management tool ZenHub, to
+provide a better overview of the project completion status.
+- The members of the group are each responsible for keeping up with the content
+of the other semester subjects.
+- The supervisor meeting is scheduled for each Friday, however the time of each
+meeting can vary and are therefor is the time agreed upon on the previous
```

Comment:
"... however, the time of each meeting can vary and is therefore agreed upon duringthe previous meeting."